app/presenters/help_page_presenter.rb

```diff
@@ -0,0 +1,2 @@
+class HelpPagePresenter < GovspeakPresenter
```

Comment:
I think GovspeakPresenter is misleading. Many formats present govspeak.

Perhaps a `body` module that has one method:
```ruby
def body
  content_item["details"]["body"]
end
```

Then the two presenters include that module. Presenters currently use modules rather than inheritance.